### DISCUSSION TITLE
Should we change the way the cards scroll on mobiles?

### DISCUSSION THREAD

**Discussion by salmonb** [webfx-project/webfx-website#5] [General]
The purpose of this discussion is to decide if we open an issue regarding the website vertical scroll on mobiles (is it intuitive enough?).

This is a continuation of this topic started in the first discussion https://github.com/webfx-project/webfx-website/discussions/1

So far we were wondering if we should switch from a swipe up/down navigation to a more standard scrollbar navigation.

What do you think?

**Comment by dukke** [comment]
BTW, I have an open source project coming up: FXSkins.
One of the skins is going to change the JavaFX scrollbar to something like you wanted and mentioned. That is, a ScrollPane with just a small thumb overlaid on top to the right. A more minimal ScrollPane, similar to the ones we see on regular websites when we browse with our phones.
